spec/features/user_sees_homepage_spec.rb

```diff
@@ -4,6 +4,41 @@
   it 'has a homepage' do
 
     visit root_path
-    expect(page).to have_content("Welcome")
+    expect(page).to have_content("Endangerous Outfitters")
   end
+
+  it 'has a login link' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.